libs/ballot-interpreter/src/options.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  AdjudicationReason,
+  ElectionDefinition,
+  MarkThresholds,
+  PrecinctSelection,
+} from '@votingworks/types';
+
+/**
+ * Options for interpreting a sheet of ballot images.
+ */
+export interface InterpreterOptions {
+  electionDefinition: ElectionDefinition;
+  precinctSelection: PrecinctSelection;
+  testMode: boolean;
+  markThresholds: MarkThresholds;
+  adjudicationReasons: readonly AdjudicationReason[];
+}
+
+export function getScoreWriteInsFlag(options: InterpreterOptions): boolean {
```

Comment:
Nit: maybe name this `shouldScoreWriteIns`

Also, do we need to make sure that `markThresholds.writeInTextArea` is defined if unmarked write-in detection is turned on? Or maybe we should just make `markThresholds.writeInTextArea` required, since the adjudication reason is now the on/off switch.

Reply:
That's a good point. I substitute in the default in the Typescript, but maybe the rust interpreter is asserting it exists if the flag is on?

This is one of the pieces that I was unsure about pending your work.

Reply:
OK. Yeah, the place where the threshold is required I just have a temporary default in place until our threshold strategy gets clearer.